server/src/main/java/com/cloud/ha/dao/HighAvailabilityDao.java

```diff
@@ -16,12 +16,12 @@
 // under the License.
 package com.cloud.ha.dao;
 
-import java.util.List;
-
```

Comment:
Same as above

Reply:
I think thats happened during the merge. My IntelliJ sorts the imports differently, for whatever reason..